controllers/ebpf/bpfmanager-controller.go

```diff
@@ -0,0 +1,240 @@
+package ebpf
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"github.com/netobserv/network-observability-operator/pkg/helper"
+	"github.com/sirupsen/logrus"
+
+	flowslatest "github.com/netobserv/network-observability-operator/apis/flowcollector/v1beta2"
+
+	bpfmaniov1alpha1 "github.com/bpfman/bpfman-operator/apis/v1alpha1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	netobservApp     = "netobserv"
+	netobservBCImage = "quay.io/netobserv/ebpf-bytecode:latest"
```

Comment:
can you make that picked up from the command line argument, same as the other images (cf in main.go `flag.StringVar(&config.EBPFAgentImage, "ebpf-agent-image", "quay.io/netobserv/netobserv-ebpf-agent:main", "The image of the eBPF agent")`